OpenRA.Mods.D2k/Traits/WormFood.cs

```diff
@@ -0,0 +1,53 @@
+#region Copyright & License Information
+/*
+* Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+* This file is part of OpenRA, which is free software. It is made
+* available to you under the terms of the GNU General Public License
+* as published by the Free Software Foundation. For more information,
+* see COPYING.
+*/
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.D2k.Traits
+{
+	[Desc("This actor attracts sandworms.")]
+	class WormFoodInfo : ITraitInfo
+	{
+		[Desc("List of terrains on which making noise can get you killed by a sandworm.")]
+		public static HashSet<string> DangerousTerrain = new HashSet<string> { "Sand", "Dune", "Spice", "SpiceBlobs" };
```

Comment:
This is inconsistent with our other traits, and I don't think or FieldLoader even understands HashSets.